OpenRA.Mods.Common/Traits/Buildings/RallyPoint.cs

```diff
@@ -40,6 +40,8 @@ public class RallyPoint : IIssueOrder, IResolveOrder, ISync, INotifyOwnerChanged
 		public RallyPointInfo Info;
 		public string PaletteName { get; private set; }
 
+		private const uint ExtraDataSetPrimaryBuilding = 1;
```

Comment:
Explicitly declaring this is better than having a magic value.  The `private` should be dropped, however, because that is already the default.